SashaZaitsev/HW3/03.built-in data types.py

```diff
@@ -0,0 +1,38 @@
+# built-in data types
+better = 0
+never = 0
+iss = 0
+
+line_zen_python = input('Введіть рядок з "Zen of Python":').lower()
+for i in line_zen_python.split():
```

Comment:
more optimal piece of your code

print("In 'Zen of Python' the word \"better\" repeat", line_zen_python.count("better"), 'times')
print("In 'Zen of Python' the word \"never\" repeat", line_zen_python.count("never"), 'times')
print("In 'Zen of Python' the word \"is\" repeat", line_zen_python.count("is"), 'times')